src/Lambda/LambdaClient.php

```diff
@@ -67,4 +69,40 @@
  * @method \Aws\Result updateFunctionConfiguration(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateFunctionConfigurationAsync(array $args = [])
  */
-class LambdaClient extends AwsClient {}
+class LambdaClient extends AwsClient
+{
+    /**
+     * Default Curl options for Lambda
+     * @var array
+     */
+    static $defaultCurlOptions = [
+        CURLOPT_TCP_KEEPALIVE => 1,
```

Comment:
This configuration should be local to the middleware generation below and behind a `curl` extension check since we don't require cURL. [See this PR and it's related bug](https://github.com/aws/aws-sdk-php/pull/1481/files#diff-85d927b5b18f9813bfd8535665c1e98dR62).

Reply:
Agreed about the `curl` extension check, but keeping this as a class variable allows re-use for testing, minimizing future code changes should options change.